rest-api-spec/src/main/resources/rest-api-spec/api/indices.render_search_template.json

```diff
@@ -1,5 +1,5 @@
 {
-  "render_search_template": {
+  "indices.render_search_template": {
```

Comment:
But nit picky perhaps but I think this actually makes more sense unnamespaced like other search related API's. Do not feel strongly about this though.

Reply:
Thanks, you're right - don't know what i was thinking